src/main/java/nl/hsac/fitnesse/slim/AllArgScenarioTable.java

```diff
@@ -23,11 +24,23 @@
 
     public AllArgScenarioTable(Table table, String tableId, SlimTestContext testContext) {
         super(table, tableId, testContext);
+        String s = "";
     }
 
     @Override
     public List<SlimAssertion> call(Map<String, String> scenarioArguments,
                                     SlimTable parentTable, int row) throws TestExecutionException {
+        /* set number of total number of tests for scenario
```

Comment:
Why do you put both values in the arguments twice (under different names)?

Reply:
I wanted a short name and long name in case someone might be using CURRENT_ROW in their decision table as actual input argument. Now I can see that I would overwrite them anyway so there is no point to this. I will leave one variable for each value here.